src/pps/pps.proto

```diff
@@ -312,8 +313,8 @@ message Worker {
 }
 
 message Pipeline {
-  string name = 1;
   pfs_v2.Project project = 2;
+  string name = 1;
```

Comment:
nit: I think it's worth sticking with the convention of proto fields being in ascending index order. When we add a lot of fields it makes it easy to figure out what the next open slots are

Reply:
That contends with the convention of putting the fields in order of relevance or scope. For example, when `repo_type` was added it was placed after `repo` rather than after `trigger`. In this case, @brycemcanally requested that the order of fields in the protos be project, repo, branch, commit in order that they go from largest to smallest scope.

Similarly, in `auth.proto` we group the `Permission` enum by related permissions.